src/cdk-experimental/clipboard/clipboard.md

```diff
@@ -0,0 +1,62 @@
+**Warning: this component is still experimental. It may have bugs and the API may change at any
+time**
+
+The clipboard package provides helpers for working with the system clipboard.
+
+## The `Clipboard` service
```

Comment:
AFAIK we aren't supposed to start headlines with "The". There's another case further down.